libcxx/test/std/atomics/atomics.ref/store.pass.cpp

```diff
@@ -19,27 +19,49 @@
 #include "test_helper.h"
 #include "test_macros.h"
 
-template <typename T>
+template <typename T, typename U>
+concept has_store = requires { std::declval<T>().store(std::declval<U>()); };
+
+template <typename U>
 struct TestStore {
   void operator()() const {
+    static_assert(has_store<std::atomic_ref<U>, U>);
+    do_test<U>();
+    do_test_atomic<U>();
+    static_assert(!has_store<std::atomic_ref<U const>, U>);
```

Comment:
It would be better to test constraints on other modifying operations.

Reply:
I meant testing other `static_assert(!has_meow<std::atomic_ref<U const>>>)` , where each `has_meow` detects one of `compare_exchange_weak`, `fetch_add`, etc..

Reply:
Oh I see.  Yes I intended to do that after we converge on the few tests that I already updated.